src/plugins/intel_gpu/tests/unit/fusions/concatenate_fusion_test.cpp

```diff
@@ -144,3 +144,130 @@ INSTANTIATE_TEST_SUITE_P(fusings_gpu, concat_onednn_eltwise, ::testing::ValuesIn
+                return true;
+            return false;
+        };
+
+        auto pi_fused = network_fused.get_primitives_info();
+        auto pi_not_fused = network_not_fused.get_primitives_info();
+
+        auto info_fused = std::find_if(pi_fused.begin(), pi_fused.end(), find_and_check);
+        auto info_not_fused = std::find_if(pi_not_fused.begin(), pi_not_fused.end(), find_and_check);
+
+        ASSERT_TRUE(info_fused != pi_fused.end());
+        ASSERT_TRUE(info_not_fused != pi_not_fused.end());
+
+        compare(network_not_fused, network_fused, p);
+    }
+
+    layout get_input_layout(concat_test_params& p) {
+        return layout{ p.data_type, p.input_format, p.in_shape };
+    }
+
+    layout get_per_channel_layout(concat_test_params& p) {
+        return layout{ p.default_type, p.default_format, tensor{ 1, p.in_shape.feature[0], 1, 1 } };
+    }
+};
+}  // namespace
+
+
+/* ----------------------------------------------------------------------------------------------------- */
+/* --------------------------------------- Concat cases ------------------------------------------------ */
+/* ----------------------------------------------------------------------------------------------------- */
+#define CASE_CONCAT_F32_1 { 1, 8, 4, 4 }, data_types::f32, format::bfyx, data_types::f32, format::bfyx
+#define CASE_CONCAT_F16_1 { 1, 8, 4, 4 }, data_types::f16, format::bfyx, data_types::f16, format::bfyx
+
+class concat_activation : public ConcatFusingTest {};
+TEST_P(concat_activation, along_f) {
+    auto p = GetParam();
+
+    if (engine.get_device_info().supports_immad)
+        p.expected_fused_primitives++;
+
+    create_topologies(
+        input_layout("input0", get_input_layout(p)),
+        input_layout("input1", get_input_layout(p)),
+        concatenation("concat", { input_info("input0"), input_info("input1") }, 1, p.data_type),
+        activation("act1", input_info("concat"), activation_func::round_half_to_even),
+        activation("act2", input_info("act1"), activation_func::clamp),
+        reorder("reorder_bfyx", input_info("act2"), cldnn::format::bfyx, p.default_type)
+    );
+
+    tolerance = 1.f;
+    execute(p);
+}
+
+class concat_eltwise : public ConcatFusingTest {};
+TEST_P(concat_eltwise, along_f) {
+    auto p = GetParam();
+    layout data_layout(p.default_type, p.default_format, tensor{ 1, p.in_shape.feature[0]*2, 1, 1 });
+
+    create_topologies(
+        input_layout("input0", get_input_layout(p)),
+        input_layout("input1", get_input_layout(p)),
+        data("scale_data", get_mem(data_layout, 1.0f / tensor{ 1, 1, 4, 4 }.count())),
+        concatenation("concat", { input_info("input0"), input_info("input1") }, 1, p.data_type),
+        eltwise("scale", { input_info("concat"), input_info("scale_data") }, eltwise_mode::prod, p.default_type),
+        reorder("reorder_bfyx", input_info("scale"), cldnn::format::bfyx, p.default_type)
+    );
+
+    tolerance = 1.f;
```

Comment:
for fp16/fp32 the threshold looks too big. Is there any reason to use 1.0 here?

Reply:
No, it's a copy-paste mistake